Specs/add_line_numbers.rb

```diff
@@ -0,0 +1,17 @@
+def puzzle_2
+  chapter2 = File.read('add_line_numbers.txt')
+  output = add_line_numbers(chapter2)
+  print output
+end
+
+def add_line_numbers(chapter2)
+  text = chapter2.split("\n")
+  index = 0
+  text_with_index = text.map do |wers|
```

Comment:
`text_with_index` will be an array with numbered lines of text. I'd call it `lines` or `text_with_numbered_lines`.

```suggestion
  lines = text.map do |wers|
```